README.md

```diff
@@ -16,6 +16,8 @@ Please see [CONTRIBUTING](CONTRIBUTING.md) for details on how to organize and co
     * [Lazy loading with react-router](javascript/react-router/lazy-loading/)
   - React-Crossfilter
     * [A simple React wrapper for crossfilter](javascript/react-crossfilter/)
+  - Piecewise-linear-scaling
```

Comment:
This line should be removed and replaced with line 20 (Piecewise-linear-scaling) isn't the name of a library/framework.